README.md

```diff
@@ -20,13 +20,13 @@ This library has been tested against the following versions of Go
 ## Install
 
 ```console
-$ go get github.com/bold-commerce/go-shopify
+$ go get github.com/bold-commerce/go-shopify/v3
```

Comment:
From my point of view we need to document how to access both versions (2.3 and 3) so it will be beneficial to add both links in the readme file rather than replace it.